### ISSUE TITLE
utime went backwards!

### ISSUE THREAD

**Issue by richvdh** [matrix-org/synapse#5608]
Observed during startup:

```
2019-07-03 08:51:31,442 - synapse.util.logcontext - 381 - ERROR - None - utime went backwards! 0.000000 < 1.108000
2019-07-03 08:51:31,445 - synapse.util.logcontext - 389 - ERROR - None - stime went backwards! 0.000000 < 0.096000
```

presumably related to #5499 / #3548

**Comment by richvdh** [comment]
only seems to happen when synapse is set to daemonize

**Comment by richvdh** [comment]
fixed by #5609
